nav2_motion_primitives/src/back_up.cpp

```diff
@@ -45,8 +45,8 @@ Status BackUp::onRun(const std::shared_ptr<const BackUpAction::Goal> command)
 
   command_x_ = command->target.x;
 
-  if (!robot_->getOdometry(initial_pose_)) {
-    RCLCPP_ERROR(node_->get_logger(), "initial robot odom pose is not available.");
+  if (!collision_checker_->getRobotPose(initial_pose_)) {
```

Comment:
Alternatively use the `GetRobotPoseClient`.

Reply:
So we duplicate the `GetRobotPoseClient` here and in the `collision_checker`?

Reply:
Based on our conversation, can we reference an issue # so we can track it later?